package.json

```diff
@@ -43,6 +43,7 @@
       "build": "npm run build:cjs && npm run build:es",
       "build:cjs": "tsc -b packages/test/tsconfig.cjs.json",
       "build:es": "tsc -b packages/test/tsconfig.es.json",
-      "test": "lerna run test --parallel"
+      "test": "lerna run test --parallel",
+      "rollup": "rollup" 
```

Comment:
Why add rollup?

Reply:
Was testing why rollup command was not getting picked up with node 14